frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segment_statistics_modal.tsx

```diff
@@ -35,21 +50,25 @@ type SegmentInfo = {
   formattedSize: string;
   volumeInVoxel: number;
   boundingBoxTopLeft: Vector3;
+  boundingBoxTopLeftAsString: string;
   boundingBoxPosition: Vector3;
+  boundingBoxPositionAsString: string;
 };
 
 const exportStatisticsToCSV = (
-  segmentInformation: Array<SegmentInfo>,
+  segmentInformation: Array<SegmentInfo> | string,
```

Comment:
`segmentInformation` will be a string only in the error case, right? I would try to avoid calling this function in the first place if an error occurred. see my other comment for details.

Reply:
I think it won't be called in an error case because the ok button is disabled in that case

Reply:
ok, then the function (and its interface) should not have to deal with the `string` case even more :)